actionlist/__tests__/components/actionItemBodySpec.js

```diff
@@ -20,7 +20,7 @@ describe('Action_Item_Body', () => {
         label: 'action requested label',
       },
       initiator: 'initiator',
-      lastApprovedDate: 'last approved date',
+      creationDate: 'creation date',
```

Comment:
These values should all be refactored out into constants since they appear in multiple places and especially since you're using them to assert correctness.